test/LibraryManager.IntegrationTest/AddCientSideLibrariesFromUITests.cs

```diff
@@ -0,0 +1,120 @@
+﻿using System.IO;
+using Microsoft.Test.Apex.VisualStudio.Shell.ToolWindows;
+using Microsoft.Test.Apex.VisualStudio.Solution;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Microsoft.Web.LibraryManager.IntegrationTest.Services;
+
+namespace Microsoft.Web.LibraryManager.IntegrationTest
+{
+    [TestClass]
+    public class AddCientSideLibrariesFromUITests : VisualStudioLibmanHostTest
```

Comment:
>m [](start = 67, length = 1)

nit: do you guys capitalize the m in Libman?

Reply:
Hmmm, I don't see that anywhere in the project. I updated one place in this file to keep it consistent

---
In reply to: [206365281](https://github.com/aspnet/LibraryManager/pull/332#discussion_r206365281) [](ancestors = 206365281)